report.py

```diff
@@ -0,0 +1,86 @@
+import os
+import numpy as np
+import pandas as pd
+import shutil
+from django.db.models import Count
+from data_refinery_common.models import Organism, Sample
+from data_refinery_common.logging import get_and_configure_logger
+
+
+logger = get_and_configure_logger(__name__)
+ROOT_DIR = "/home/user/data_store/"
+
+
+def organism_can_have_qn_target(organism: Organism, sample_threshold=100):
+    """ Check that the organism has more than `sample_threshold` samples on
+    some microarray platform """
+    microarray_platforms = organism.sample_set\
+        .filter(has_raw=True, technology="MICROARRAY", is_processed=True)\
+        .values('platform_accession_code')\
+        .annotate(count=Count('id'))\
+        .filter(count__gt=sample_threshold)
+
+    return microarray_platforms.exists()
+
+
+organism_name = "DANIO_RERIO"
```

Comment:
My plan is to manually change this to HOMO_SAPIENS, MUS_MUSCULUS, or None depending on which set I'm trying to run.